docs/performance_data.md

```diff
@@ -0,0 +1,21 @@
+Performance Data
+================
+
+AWX has the ability to collect performance data on job runs.
+
+The following data is collected periodically (by default, every 0.25 seconds):
+* CPU usage
+* Memory usage
+* PID count
+
+The data is stored under `/var/log/tower/playbook_profiling`. A new folder is created for each job run. The folder's name is set to the job's ID.
```

Comment:
how does the data get cleaned up?

Reply:
We discussed this during the merge meeting and decided against adding a system job. So, currently, clean-up has to be done manually:

```
ansible instances -i cluster -m file -a "state=absent src=/var/log/tower/playbook_profiling"
```